pkgs/test/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 1.26.1-wip
 
 * Set a debug name for test isolates.
+* Use version `0.7.6` of `packge:test_api` to fix an assertion failure when
```

Comment:
nit: just copy the changelog from test_api directly - we pretend as if these are all just one package.